### ISSUE TITLE
Animations fail with gsap not defined

### ISSUE THREAD

**Issue by davidR1974** [gsimon2/dramatic-rolls#34]
Foundry : 11.312 & Foudnry 12.329
system : dnd5e (3.3.1)

Some animations (Number Pop Critical, Number Fly, Number Font Switch) fail with following error :

```
Uncaught ReferenceError: gsap is not defined
    at numberPop (numberAnimations.js:76:15)
    at Animation.animationFunction (animationController.js:37:13)
    at Animation.play (animationController.js:27:15)
    at AnimationController.playById (animationController.js:135:20)
    at ConfigureAnimationSettingsForm.onPlayAnimation (configureAnimationsSettingsForm.js:100:27)
    at ConfigureAnimationSettingsForm.<anonymous> (configureAnimationsSettingsForm.js:66:29)
    at HTMLButtonElement.dispatch (jquery.min.js:2:43184)
    at y.handle (jquery.min.js:2:41168)
```

adding the line :
```
import gsap from "/scripts/greensock/esm/all.js";
```

at start of numberAnimations.js

seems to resolve the issue (animations play wothout error)

**Comment by gsimon2** [comment]
Yea that makes sense. 

Not really sure why it worked for all the development and testing 🤷‍♂️ Even did a clean install of the module before release.

Will push a fix out soon. Thanks for reporting it!

**Comment by gsimon2** [comment]
Should be fixed in version `3.0.1`. Thanks for bringing the issue to my attention.

**Comment by davidR1974** [comment]
was running on Foundry Node version (with node version 20.13.1). don't know if that can help to explain differences  with testing.
